### PR TITLE
Release notes for prerelease package

### PR DESCRIPTION
**Pull request by nishitha-burman**
null

AB#43027087

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
+* ICoreWebView2ExperimentalProfile6
+   SetPermissionState
+   GetNonDefaultPermissionSettings
+* ICoreWebView2ExperimentalPermissionSettingCollectionView
+   GetValueAtIndex
+   get_Count
+* ICoreWebView2ExperimentalPermissionSetting
+   get_PermissionKind
+   get_PermissionOrigin
+   get_PermissionState
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:173]:
```suggestion
* [ICoreWebView2ExperimentalPermissionRequestedEventArgs3 interface](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalpermissionrequestedeventargs3?view=webview2-1.0.1619-prerelease&preserve-view=true)
   * [ICoreWebView2ExperimentalPermissionRequestedEventArgs3::SavesInProfile property (get](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalpermissionrequestedeventargs3?view=webview2-1.0.1619-prerelease&preserve-view=true#get_savesinprofile), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalpermissionrequestedeventargs3?view=webview2-1.0.1619-prerelease&preserve-view=true#put_savesinprofile)
* [ICoreWebView2ExperimentalSetPermissionStateCompletedHandler interface](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsetpermissionstatecompletedhandler?view=webview2-1.0.1619-prerelease&preserve-view=true)
* [ICoreWebView2ExperimentalGetNonDefaultPermissionSettingsCompletedHandler interface](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalgetnondefaultpermissionsettingscompletedhandler?view=webview2-1.0.1619-prerelease&preserve-view=true)
* [ICoreWebView2ExperimentalProfile6 interface](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalprofile6?view=webview2-1.0.1619-prerelease&preserve-view=true)
   * [ICoreWebView2ExperimentalProfile6::GetNonDefaultPermissionSettings method](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalprofile6?view=webview2-1.0.1619-prerelease&preserve-view=true#getnondefaultpermissionsettings)
   * [ICoreWebView2ExperimentalProfile6::SetPermissionState method](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalprofile6?view=webview2-1.0.1619-prerelease&preserve-view=true#setpermissionstate)
* [ICoreWebView2ExperimentalPermissionSettingCollectionView interface](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalpermissionsettingcollectionview?view=webview2-1.0.1619-prerelease&preserve-view=true)
   * [ICoreWebView2ExperimentalPermissionSettingCollectionView::Count property (get)](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalpermissionsettingcollectionview?view=webview2-1.0.1619-prerelease&preserve-view=true#get_count)<!--no put-->
   * [ICoreWebView2ExperimentalPermissionSettingCollectionView::GetValueAtIndex method](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalpermissionsettingcollectionview?view=webview2-1.0.1619-prerelease&preserve-view=true#getvalueatindex)
* [ICoreWebView2ExperimentalPermissionSetting interface](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalpermissionsetting?view=webview2-1.0.1619-prerelease&preserve-view=true)
   * [ICoreWebView2ExperimentalPermissionSetting::PermissionKind property (get)](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalpermissionsetting?view=webview2-1.0.1619-prerelease&preserve-view=true#get_permissionkind)<!--no put-->
   * [ICoreWebView2ExperimentalPermissionSetting::PermissionOrigin property (get)](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalpermissionsetting?view=webview2-1.0.1619-prerelease&preserve-view=true#get_permissionorigin)<!--no put-->
   * [ICoreWebView2ExperimentalPermissionSetting::PermissionState property (get)](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalpermissionsetting?view=webview2-1.0.1619-prerelease&preserve-view=true#get_permissionstate)<!--no put-->
```

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
 
 ##### [Win32/C++](#tab/win32cpp)
 
-* 
+* ICoreWebView2EnvironmentOptions5
+   get_EnableTrackingPrevention
+   put_EnableTrackingPrevention
+* ICoreWebView2Profile3
+   get_PreferredTrackingPreventionLevel
+   put_PreferredTrackingPreventionLevel
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:250]:
```suggestion
* [ICoreWebView2EnvironmentOptions5 interface](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions5?view=webview2-1.0.1619-prerelease&preserve-view=true)
   * [ICoreWebView2EnvironmentOptions5::EnableTrackingPrevention property (get](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions5?view=webview2-1.0.1619-prerelease&preserve-view=true#get_enabletrackingprevention), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions5?view=webview2-1.0.1619-prerelease&preserve-view=true#put_enabletrackingprevention)
* [ICoreWebView2Profile3 interface](/microsoft-edge/webview2/reference/win32/icorewebview2profile3?view=webview2-1.0.1619-prerelease&preserve-view=true)
   * [ICoreWebView2Profile3::PreferredTrackingPreventionLevel property (get](/microsoft-edge/webview2/reference/win32/icorewebview2profile3?view=webview2-1.0.1619-prerelease&preserve-view=true#get_preferredtrackingpreventionlevel), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2profile3?view=webview2-1.0.1619-prerelease&preserve-view=true#put_preferredtrackingpreventionlevel)
```

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
 
 ---
 
 #### Bug fixes
  
-*   -->
+* Disable "Open link as Profile" in the WebView2 context menu.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:256]:
```suggestion
*  Disable **Open link as Profile** in the WebView2 context menu.
```

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
+* Fix post data missing in form submit with control click. ([Issue #2652](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2652))
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:257]:
```suggestion
*  Fix post data missing in form submit with control-click. ([Issue #2652](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2652))
```

File: microsoft-edge/webview2/release-notes.md [hunk 5]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
+* Fixed a bug where the user is not able to get the custom context menu on PDF Viewer. ([Issue #2607](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2607))
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:258]:
```suggestion
*  Fixed a bug where the user is not able to get the custom context menu on PDF Viewer. ([Issue #2607](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2607))
```

File: microsoft-edge/webview2/release-notes.md [hunk 6]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
+* Fixed bug where the whole tool bar will be blank when hiding Bookmarks, Search, and PageSelector buttons simultaneously. ([Issue #2866](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2866))
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:259]:
```suggestion
*  Fixed bug where the entire tool bar is blank when simultaneously hiding the **Bookmarks**, **Search**, and **PageSelector** buttons. ([Issue #2866](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2866))
```

File: microsoft-edge/webview2/release-notes.md [hunk 7]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
+* Fixed a bug where the app crashes when trying to move focus to WebView2 when it is disabled.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:260]:
```suggestion
*  Fixed a bug where the app crashes when trying to move focus to WebView2 when it is disabled.
```

File: microsoft-edge/webview2/release-notes.md [hunk 8]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
+* Fix drag/drop within the WebView2 for composition hosted WebViews.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:261]:
```suggestion
*  Fix drag-and-drop within the WebView2 for composition-hosted WebViews.
```

File: microsoft-edge/webview2/release-notes.md [hunk 9]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
+* Remove read aloud icon in address bar in a WebView2 popup window.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:262]:
```suggestion
*  Removed read-aloud icon in address bar in a WebView2 popup window.
```

File: microsoft-edge/webview2/release-notes.md [hunk 10]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
+* Fix unexpected items in context menu of popup windows in WebView2.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:263]:
```suggestion
*  Fixed unexpected items in the context menu of popup windows in WebView2.
```

File: microsoft-edge/webview2/release-notes.md [hunk 11]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
 
 ##### [.NET/C#](#tab/dotnetcsharp)
 
-* 
+* * CoreWebView2PermissionRequestedEventArgs Class
+   CoreWebView2PermissionRequestedEventArgs.SaveInProfile Property
+* ICoreWebView2PermissionSetting Class
+   ICoreWebView2PermissionSetting.PermissionKind Property
+   ICoreWebView2PermissionSetting.PermissionOrigin Property
+   ICoreWebView2PermissionSetting.PermissionState Property
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:146]:
```suggestion
* [CoreWebView2PermissionRequestedEventArgs Class](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionrequestedeventargs?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true&preserve-view=true)
   * [CoreWebView2PermissionRequestedEventArgs.SavesInProfile Property](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionrequestedeventargs.savesinprofile?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true)
* [CoreWebView2PermissionSetting Class](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionsetting?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true)
   * [CoreWebView2PermissionSetting.PermissionKind Property](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionsetting.permissionkind?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true)
   * [CoreWebView2PermissionSetting.PermissionOrigin Property](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionsetting.permissionorigin?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true)
   * [CoreWebView2PermissionSetting.PermissionState Property](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionsetting.permissionstate?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true)
```

File: microsoft-edge/webview2/release-notes.md [hunk 12]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
 
 ---
 
-*  Added support for the tracking prevention API:
+*  Added support for API to disable back and forward navigation:
 
 ##### [.NET/C#](#tab/dotnetcsharp)
 
-* 
+* CoreWebView2NavigationStartingEventArgs
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:181]:
```suggestion
* [CoreWebView2NavigationStartingEventArgs Class](/dotnet/api/microsoft.web.webview2.core.corewebview2navigationstartingeventargs?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true)
```

File: microsoft-edge/webview2/release-notes.md [hunk 13]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
 
-*  Added support for the Print API:
+*  The Custom Scheme Registration API:
 
 ##### [.NET/C#](#tab/dotnetcsharp)
 
-* 
+* CoreWebView2EnvironmentOptions Class
+   CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property
+* CoreWebView2CustomSchemeRegistration Class
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:203]:
```suggestion
* [CoreWebView2EnvironmentOptions Class](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true)
   * [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.customschemeregistrations?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true)
* [CoreWebView2CustomSchemeRegistration Class](/dotnet/api/microsoft.web.webview2.core.corewebview2customschemeregistration?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true)
```

File: microsoft-edge/webview2/release-notes.md [hunk 14]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
 
 ##### [WinRT/C#](#tab/winrtcsharp)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:206]: @nishitha-burman @peiche-jessica - property not found, for WinRT:
CoreWebView2EnvironmentOptions.**CustomSchemeRegistrations** Property

Class:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions?view=webview2-winrt-1.0.1619-prerelease

It's found for .NET:
* [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.customschemeregistrations?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true)

Equiv is found for Win32:
   * [ICoreWebView2EnvironmentOptions4::GetCustomSchemeRegistrations method](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions4?view=webview2-1.0.1619-prerelease&preserve-view=true#getcustomschemeregistrations)
   * [ICoreWebView2EnvironmentOptions4::SetCustomSchemeRegistrations method](https://learn.microsoft.com//microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions4?view=webview2-1.0.1619-prerelease&

File: microsoft-edge/webview2/release-notes.md [hunk 15]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
 
-* 
+* CoreWebView2PermissionRequestedEventArgs Class
+   CoreWebView2PermissionRequestedEventArgs.SaveInProfile Property
+* ICoreWebView2PermissionSetting Class
+   ICoreWebView2PermissionSetting.PermissionKind Property
+   ICoreWebView2PermissionSetting.PermissionOrigin Property
+   ICoreWebView2PermissionSetting.PermissionState Property
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:155]:
```suggestion
* [CoreWebView2PermissionRequestedEventArgs Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionrequestedeventargs?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true)
   * [CoreWebView2PermissionRequestedEventArgs.SavesInProfile Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionrequestedeventargs?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true#savesinprofile)
* [CoreWebView2PermissionSetting Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionsetting?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true)
   * [CoreWebView2PermissionSetting.PermissionKind Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionsetting?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true#permissionkind)
   * [CoreWebView2PermissionSetting.PermissionOrigin Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionsetting?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true#permissionorigin)
   * [CoreWebView2PermissionSetting.PermissionState Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionsetting?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true#permissionstate)
```

File: microsoft-edge/webview2/release-notes.md [hunk 16]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
 
 ##### [.NET/C#](#tab/dotnetcsharp)
 
-* 
+* CoreWebView2NavigationStartingEventArgs
 
 ##### [WinRT/C#](#tab/winrtcsharp)
 
-* 
+* CoreWebView2NavigationStartingEventArgs
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:185]:
```suggestion
* [CoreWebView2NavigationStartingEventArgs Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2navigationstartingeventargs?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true)
```

File: microsoft-edge/webview2/release-notes.md [hunk 17]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
+* CoreWebView2EnvironmentOptions Class
+   CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property
+* CoreWebView2CustomSchemeRegistration Class
 
 ##### [WinRT/C#](#tab/winrtcsharp)
 
-* 
+* CoreWebView2EnvironmentOptions Class
+   CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property
+* CoreWebView2CustomSchemeRegistration Class
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:209]:
```suggestion
* [CoreWebView2EnvironmentOptions Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true)
* [CoreWebView2CustomSchemeRegistration Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2customschemeregistration?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true)
```

File: microsoft-edge/webview2/release-notes.md [hunk 18]
```diff
@@ -122,89 +122,145 @@ The following items are now stable:
...
+* CoreWebView2Profile Class
+   CoreWebView2Profile.PreferredTrackingPreventionLevel Property
 
 ##### [WinRT/C#](#tab/winrtcsharp)
 
-* 
+* CoreWebView2EnvironmentOptions Class
+   CoreWebView2EnvironmentOptions.EnableTrackingPrevention Property
+* CoreWebView2Profile Class
+   CoreWebView2Profile.PreferredTrackingPreventionLevel Property
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:241]:
```suggestion
* [CoreWebView2EnvironmentOptions Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true)
   * [CoreWebView2EnvironmentOptions.EnableTrackingPrevention Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true#enabletrackingprevention)
* [CoreWebView2Profile Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true)
   * [CoreWebView2Profile.PreferredTrackingPreventionLevel Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true#preferredtrackingpreventionlevel)
```

File: microsoft-edge/webview2/release-notes.md (hunk 7 above)
Comment (nishitha-burman) [reply microsoft-edge/webview2/release-notes.md]: When the WebView is disabled

Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md]:
```suggestion
* Fixed a bug where the app crashes when trying to move focus to WebView2 when the WebView2 control is disabled.
```

**Review by mikehoffms — APPROVED** [review]

**Comment by JasonAndrewWriter** [comment]
rendered: https://github.com/MicrosoftDocs/edge-developer/blob/bfd64daa6f44cf31fd59cd25dd4b8b69193cec74/microsoft-edge/webview2/release-notes.md

**Comment by mikehoffms** [comment]
Rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-2411

GH rendered (manually nav'd, so not commit-specific):
https://github.com/MicrosoftDocs/edge-developer/blob/user/niburman/ReleaseNotes109Prerelease/microsoft-edge/webview2/release-notes.md

**Review by JasonAndrewWriter — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md [hunk 19]
```diff
@@ -122,89 +122,157 @@ The following items are now stable:
...
+* [CoreWebView2PermissionRequestedEventArgs Class](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionrequestedeventargs?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true&preserve-view=true)
+   * [CoreWebView2PermissionRequestedEventArgs.SavesInProfile Property](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionrequestedeventargs.savesinprofile?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true)
+* [CoreWebView2PermissionSetting Class](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionsetting?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true)
+   * [CoreWebView2PermissionSetting.PermissionKind Property](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionsetting.permissionkind?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true)
+   * [CoreWebView2PermissionKind Enum](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2permissionkind?view=webview2-dotnet-1.0.1619-prerelease)
+      * MultipleAutomaticDownloads
+      * FileReadWrite
+      * Autoplay
+      * LocalFonts
+      * MidiSystemExclusiveMessageAccess
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:150]:
```suggestion
   * [CoreWebView2PermissionKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionkind?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true)
      * `MultipleAutomaticDownloads`
      * `FileReadWrite`
      * `Autoplay`
      * `LocalFonts`
      * `MidiSystemExclusiveMessageAccess`
```

File: microsoft-edge/webview2/release-notes.md [hunk 20]
```diff
@@ -122,89 +122,157 @@ The following items are now stable:
...
+   * [CoreWebView2PermissionRequestedEventArgs.SavesInProfile Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionrequestedeventargs?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true#savesinprofile)
+* [CoreWebView2PermissionSetting Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionsetting?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true)
+   * [CoreWebView2PermissionSetting.PermissionKind Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionsetting?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true#permissionkind)
+   * [CoreWebView2PermissionKind Enum](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionkind?view=webview2-winrt-1.0.1619-prerelease)
+      * MultipleAutomaticDownloads
+      * FileReadWrite
+      * Autoplay
+      * LocalFonts
+      * MidiSystemExclusiveMessageAccess
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:165]:
```suggestion
   * [CoreWebView2PermissionKind Enum](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionkind?view=webview2-winrt-1.0.1619-prerelease&preserve-view=true)
      * `MultipleAutomaticDownloads`
      * `FileReadWrite`
      * `Autoplay`
      * `LocalFonts`
      * `MidiSystemExclusiveMessageAccess`
```

File: microsoft-edge/webview2/release-notes.md [hunk 21]
```diff
@@ -122,89 +122,157 @@ The following items are now stable:
...
+   * [ICoreWebView2ExperimentalPermissionSettingCollectionView::GetValueAtIndex method](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalpermissionsettingcollectionview?view=webview2-1.0.1619-prerelease&preserve-view=true#getvalueatindex)
+* [ICoreWebView2ExperimentalPermissionSetting interface](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalpermissionsetting?view=webview2-1.0.1619-prerelease&preserve-view=true)
+   * [ICoreWebView2ExperimentalPermissionSetting::PermissionKind property (get)](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalpermissionsetting?view=webview2-1.0.1619-prerelease&preserve-view=true#get_permissionkind)<!--no put-->
+   * [COREWEBVIEW2_PERMISSION_KIND Enum](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/webview2-idl?view=webview2-1.0.1619-prerelease#corewebview2_permission_kind)
+      * COREWEBVIEW2_PERMISSION_KIND_MULTIPLE_AUTOMATIC_DOWNLOADS
+      * COREWEBVIEW2_PERMISSION_KIND_FILE_READ_WRITE
+      * COREWEBVIEW2_PERMISSION_KIND_AUTOPLAY
+      * COREWEBVIEW2_PERMISSION_KIND_LOCAL_FONTS
+      * COREWEBVIEW2_PERMISSION_KIND_MIDI_SYSTEM_EXCLUSIVE_MESSAGE_ACCESS
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:188]:
```suggestion
   * [COREWEBVIEW2_PERMISSION_KIND Enum](/microsoft-edge/webview2/reference/win32/webview2-idl?view=webview2-1.0.1619-prerelease&preserve-view=true#corewebview2_permission_kind)
      * `COREWEBVIEW2_PERMISSION_KIND_MULTIPLE_AUTOMATIC_DOWNLOADS`
      * `COREWEBVIEW2_PERMISSION_KIND_FILE_READ_WRITE`
      * `COREWEBVIEW2_PERMISSION_KIND_AUTOPLAY`
      * `COREWEBVIEW2_PERMISSION_KIND_LOCAL_FONTS`
      * `COREWEBVIEW2_PERMISSION_KIND_MIDI_SYSTEM_EXCLUSIVE_MESSAGE_ACCESS`
```

**Pull request merged**